components/support/webextensions/src/test/java/mozilla/components/support/webextensions/WebExtensionSupportTest.kt

```diff
@@ -114,12 +114,9 @@ class WebExtensionSupportTest {
             val interactionFact = facts[0]
             assertEquals(FactsAction.INTERACTION, interactionFact.action)
             assertEquals(Component.SUPPORT_WEBEXTENSIONS, interactionFact.component)
-            assertEquals(WEB_EXTENSIONS_INITIALIZED, interactionFact.item)
-            assertEquals(2, interactionFact.metadata?.size)
+            assertEquals(WEB_EXTENSION_INSTALLED, interactionFact.item)
+            assertEquals(1, interactionFact.metadata?.size)
             assertTrue(interactionFact.metadata?.containsKey("installed")!!)
-            assertTrue(interactionFact.metadata?.containsKey("enabled")!!)
-            assertEquals(listOf(ext1.id, ext2.id), interactionFact.metadata?.get("installed"))
-            assertEquals(listOf(ext1.id), interactionFact.metadata?.get("enabled"))
```

Comment:
Should we keep this line but modify it so it checks for the id?

Reply:
Is there a way to test for "WEB_EXTENSION_ENABLED"?

Reply:
The id is no longer necessary, since that was what was differentiating the `installed` fact from `enabled`. I should add a test for enabled though :D